### DISCUSSION TITLE
Introducing Play with AST page

### DISCUSSION THREAD

**Discussion by llaville** [rectorphp/rector#8759] [General]
Hello,

Two months ago, you've introducing the interactive page https://getrector.com/ast.

I would like here to share with community that there is another cool tool that exists to see/interact with AST : 
https://php-ast-viewer.com/

Project is on GitHub at https://github.com/RightCapitalHQ/php-ast-viewer

Have fun !

**Comment by TomasVotruba** [comment]
Hey, I think it was already retweeted as a reply :)

Thanks though :+1:
